data-structures/src/test/java/com/github/vmalaya/sigmasoftware/internship/datastructures/list/arraylist/InitializationTest.java

```diff
@@ -0,0 +1,55 @@
+package com.github.vmalaya.sigmasoftware.internship.datastructures.list.arraylist;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.*;
+
+@RunWith(JUnit4.class)
+public class InitializationTest {
+
+    @Test
+    public void should_create_empty_array_list (){
+
+        // given
+        List arrayList;
+        int initialCapacity = 10;
+
+        // when
+        arrayList = new ArrayList();
+        int size = arrayList.size();
+
+        // then
+        Assert.assertEquals(0,size);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void should__fail_init_array_list_by_given_capacity (){
+
+        //given
```

Comment:
please use same code style even for comments, I mean for example lines 24 and 31 use space everywhere or not use it but also everywhere